sky/clouds/cloud.py

```diff
@@ -180,7 +180,8 @@ def get_accelerators_from_instance_type(
         raise NotImplementedError
 
     @classmethod
-    def get_default_instance_type(cls) -> str:
+    def get_default_instance_type(cls,
+                                  cpu: Optional[str] = None) -> Optional[str]:
```

Comment:
Document why this would be None for some Cloud? And what happens in that case?

Reply:
If `cpu` is None, the return value will never be None. The return values can be None when `cpu` is set to an infeasible number (e.g., '123' or '1000+').

Reply:
Added the docstring. PTAL.